### ISSUE TITLE
On merge member add ability to search for members also by organization

### ISSUE THREAD

**Issue by joanagmaia** [CrowdDotDev/crowd.dev#1590]
- Change placeholder in merge member modal to "Search for members or organizations"
- Create separate endpoint which will support searching members by member displayName and organization name.
- Remove populating organizations in member/autocomplete endpoint so we don't slow down other member searches

**Comment by TheOjasSingh** [comment]
Hi I wanted to work on this issue can you assign me this issue

**Comment by TheOjasSingh** [comment]
@joanagmaia Can you help me setup crowd.dev on my local machine

**Comment by joanagmaia** [comment]
Hey @TheOjasSingh, have you checked https://github.com/CrowdDotDev/crowd.dev/blob/main/CONTRIBUTING.md?
If so, did you face any issue while setting it up?

**Comment by TheOjasSingh** [comment]
yes, when I am trying to clone the repo it is asking for permission.
![Screenshot 2023-10-10 205104](https://github.com/CrowdDotDev/crowd.dev/assets/55990400/291cf03f-ab4a-44ff-ab0e-34c14f432294)

**Comment by joanagmaia** [comment]
> yes, when I am trying to clone the repo it is asking for permission. ![Screenshot 2023-10-10 205104](https://user-images.githubusercontent.com/55990400/273968817-291cf03f-ab4a-44ff-ab0e-34c14f432294.png)

You will need to fork the repo and make a PR from there. Did you try that way?

**Comment by TheOjasSingh** [comment]
Yes I also tried that way but still when I am trying to commit it ask for permission.

**Comment by peoray** [comment]
@TheOjasSingh this might be helpful for you
https://docs.github.com/en/get-started/quickstart/fork-a-repo

**Comment by peoray** [comment]
@TheOjasSingh any success?
